### ISSUE TITLE
Batch file for easy merging maps

### ISSUE THREAD

**Issue by Outerbeast** [wootguy/bspguy#103]
To make life easier for merging maps together
User would be prompted to put in a filename for the merged bsp and then the list of bsps they wish to merge
There is an additional prompt to set a merge command which is optional
This would be run in the same dir as bspguy + to-be-merged bsps
Included a batch file that I have created before for this purpose

[bspguy_merge.zip](https://github.com/wootguy/bspguy/files/10155932/bspguy_merge.zip)

**Comment by wootguy** [comment]
The whole editor was planned to be designed around map merging, so you could place them how you want in 3D. I think the latest code or unreal's fork has a widget for merging.

**Comment by UnrealKaraulov** [comment]
@wootguy yes, in latest version possible to add multiple maps, via merge widget, paths can be entered via clipboard or using filebrowser.

But I thin it cause crashes while merging  ;) Possible I can fix it in next commits.

**Comment by wootguy** [comment]
Did something similar with `File -> Merge Multiple`. Select multiple maps, choose options, then `Merge` to replace the current map with the merged map. If the merger fails to pack maps inside the max map boundaries (likely with Half-Life's +/-4096 limit), then you get a special editor mode for arranging the maps yourself.
